tools/src/launcher/Main.hs

```diff
@@ -338,13 +346,31 @@ main =
                     loNodeLogConfig
                     (NodeData loNodePath realNodeArgs loNodeLogPath)
                     (NodeData wpath loWalletArgs loWalletLogPath)
-                    (UpdaterData
-                        loUpdaterPath loUpdaterArgs loUpdateWindowsRunner loUpdateArchive)
+                    (setUpdaterData lo)
                     loNodeTimeoutSec
                     loReportServer
                     loWalletLogging
                 logNotice "Finished clientScenario"
   where
+    setUpdaterData :: LauncherOptions -> UpdaterData
```

Comment:
Why is it called `set`? :thinking: 
I associate `set` with `Control.Lens.set`. Why not `createUpdaterData` or `constructUpdaterData` or `toUpdaterData` or `mkUpdaterData`?